Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3
+
+RUN pip install --upgrade pip
+
+WORKDIR /app
+
+COPY . .
+
+RUN pip install -r requirements.txt
+
+CMD ["python", "hacksquad_discordpy/main.py", "&"]
```

Comment:
What's the "&" for? Just wondering, that might be unnecessary.

Reply:
Its for running a command without o/p, but yeah prolly can remove it